spec/system/search/display_comments_search_spec.rb

```diff
@@ -0,0 +1,19 @@
+require "rails_helper"
+
+RSpec.describe "Display articles search spec", type: :system, js: true, elasticsearch: "FeedContent" do
+  before do
+    stub_request(:post, "http://www.google-analytics.com/collect")
```

Comment:
I started to see this around the testing failures sometimes. I thought we didn't have GA anymore :-|

Reply:
This always pops up for me locally, I am not sure why, maybe something is enabled still/"cached" locally despite us getting rid of it in the main app... its one of those easy fixes so no one has really investigated much I think

Reply:
It could be that the Google JS script has changed and now posts to that URL. I can't wait to get rid of GA :D